xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h

```diff
@@ -146,6 +146,15 @@ class CDVDInputStream
     virtual void SetVideoResolution(int width, int height) {};
   };
 
+  class IStreams
```

Comment:
that does not fit here. we deal with streams a demuxer level but not at this stage

Reply:
But not for dvds and blurays. What if I move it to IMenus?

Reply:
ok, try it.